LensGalleyPlugin.inc.php

```diff
@@ -199,12 +199,16 @@ function _getXMLContents($request, $galley) {
 		$contents = $fileService->fs->read($file->path);
 
 		// Replace media file references
-                $embeddableFilesIterator = Services::get('submissionFile')->getMany([
-                        'assocTypes' => [ASSOC_TYPE_SUBMISSION_FILE],
-                        'assocIds' => [$submissionFile->getId()],
-                        'fileStages' => [SubmissionFile::SUBMISSION_FILE_DEPENDENT],
-                        'includeDependentFiles' => true,
-		]);
+        $collector = Repo::submissionFiles()
```

Comment:
It looks like you're mixing tabs and spaces for indentation; could you set your editor to detect the indentation style and preserve it? I'd like us to convert plugins over to spaces for consistency but that should be done in its own PR (to keep cherry-picks between old- and new-style formatting as easy as possible).